src/components/show-edit-pod/show-edit-pod.component.js

```diff
@@ -19,182 +17,189 @@ const marginPropTypes = filterStyledSystemMarginProps(
   styledSystemPropTypes.space
 );
 
-class ShowEditPod extends React.Component {
-  state = {
-    editing: false,
-  };
+const ShowEditPod = ({
+  border,
+  className,
+  children,
+  saving,
+  editFields,
+  editing,
+  onEdit,
+  onSave,
+  onCancel,
+  onUndo,
+  cancel,
+  cancelText,
+  saveText,
+  onDelete,
+  deleteText,
+  softDelete,
+  buttonAlign,
+  transitionName,
+  title,
+  hideDeleteButtonInViewMode,
+  ...rest
+}) => {
+  const locale = useContext(LocaleContext);
 
-  isControlled = this.props.editing !== undefined;
+  const ref = useRef();
 
-  componentDidMount() {
-    if (this.props.editing) {
-      this.__focusOnPod();
-    }
-  }
+  const [isEditing, setIsEditingState] = useState(false);
 
-  isEditing() {
-    return this.isControlled ? this.props.editing : this.state.editing;
-  }
+  const isControlled = editing !== undefined;
 
-  onEdit = (ev) => {
-    this.props.onEdit(ev);
-    this.toggleEditingState(true);
-    this.__focusOnPod();
+  const focusPod = () => {
+    ReactDOM.findDOMNode(ref.current).focus(); // eslint-disable-line react/no-find-dom-node
```

Comment:
Why can't we just do `ref.current.focus()` here rather than using `findDOMNode`?